yalexs/activity.py

```diff
@@ -29,6 +34,7 @@
 ACTION_DOOR_OPEN_2 = "door_open"
 ACTION_DOOR_CLOSED = "doorclosed"
 ACTION_DOOR_CLOSE_2 = "door_close"
+ACTION_DOOR_CLOSE_3 = "homekit_lock"
```

Comment:
Is this a lock operation or a door operation ?

Reply:
you're right I think it's a lock operation, I moved it in lock operations dict